src/components/portifolio-header/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import "./style.css";
+
+export default function Header() {
+  return (
+    <>
+      <div className="top">
+        <a className="link" href="#home">
+          Inicio
+        </a>
+        <a className="link" href="#projects">
+          Projetos
+        </a>
+        <a className="link" href="#about">
+          Sobre
+        </a>
+        <a className="link" href="#stacks">
+          Tecnologias
+        </a>
+      </div>
+    </>
+  );
```

Comment:
Sugiro remover componente de fragmentação, pois ele não precisa ser utilizado aqui, ja que ele é usado para quando é preciso retornar duas tags html/componentes que não são 'pai'.
```suggestion
  return (
      <div className="top">
        <a className="link" href="#home">
          Inicio
        </a>
        <a className="link" href="#projects">
          Projetos
        </a>
        <a className="link" href="#about">
          Sobre
        </a>
        <a className="link" href="#stacks">
          Tecnologias
        </a>
      </div>
  );
```